kie-remote/pom.xml

```diff
@@ -13,8 +13,8 @@
 
   <dependencies>
       <dependency>
-          <groupId>org.apache.kafka</groupId>
-          <artifactId>kafka-clients</artifactId>
```

Comment:
How is it possible that `kie-remote` doesn't depend on Kafka any more?

Reply:
Ok, restored KaflaListenerThread and EvenProducer into kie-remote